test/client/spec/spec.progressive-reveal.js

```diff
@@ -49,7 +49,7 @@ describe('Progressive Reveal', function () {
 
         });
 
-        describe('groups', function () {
```

Comment:
because they're not groups of checkboxes it's just multiple ones. I don't want it to get confused with groups of radios.